src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -357,11 +357,18 @@ $pf-v6-c-tree-view--MaxNesting: 10 !default;
 
 .#{$tree-view}__list-item {
   .#{$tree-view}__list {
+    max-height: 0;
+    visibility: hidden;
     opacity: var(--#{$tree-view}__list--Opacity);
+    transition-delay: 0s, 0s, var(--#{$tree-view}__list--TransitionDuration--fade), var(--#{$tree-view}__list--TransitionDuration--fade);
```

Comment:
If on close, the `opacity` and `translate` are completed at `100ms` is a delay needed for `visibility` and `max-height`?  When setting them all to `0s` in Chrome dev tools, it looks to function normally.